docs/reference/coordinates/index.rst

```diff
@@ -1,5 +1,3 @@
-.. _sunpy-coordinates:
```

Comment:
I think this section still needs an anchor

Reply:
Fixed.